Gemfile

```diff
@@ -2,3 +2,4 @@ source 'http://rubygems.org'
 
 gem 'stix_schema_spy', '>= 1.2.3' # Used for the documentation auto-generation
 gem 'github-pages'             # Used to post to github pages
+gem 'jekyll-redirect-from' # for meta redirects
```

Comment:
The `github-pages` gem [includes](https://pages.github.com/versions/) the `jekyll-redirect-from` gem. Is this necessary?